CONTRIBUTING.md

```diff
@@ -51,6 +51,24 @@ Note that the test suite needs to pass for your changes to be accepted, so it's
 
 Even though we'd like to automate as much as possible, a comprehensive manual test is never a bad idea, especially if you're working in an area of the codebase that's particularly business critical. Reviewers are also encouraged to put UI changes through their paces, to act as a last line of defense before merging.
 
+### Formatting
+
+Code format is controlled by [Prettier](https://prettier.io/docs/en/).
+
+To format your code:
+
+```bash
+$ yarn format
```

Comment:
The readme currently uses `npm` not `yarn`, be good to have these be consistent

Reply:
I should have checked this. We use a `yarn.lock` file so assumed we use `yarn` run examples. Will fix